generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,49 @@
+const Generator = require("yeoman-generator");
+const lodash = require("lodash");
+
+module.exports = class extends Generator {
+  start() {
+    this.prompt({
+      type: "input",
+      name: "objClassName",
+      message: "Enter the name of the Obj (e.g. MyPage):",
+    }).then(answers => {
+      const objClassName = answers.objClassName;
+      const humanFriendlyName = lodash.startCase(objClassName);
+      this._writeXObjComponent(objClassName);
+      this._writeXEditingConfig(objClassName, humanFriendlyName);
+      this._writeXObjClass(objClassName);
+    });
+  }
+
+  _writeXObjComponent(objClassName) {
+    this.fs.copyTpl(
+      this.templatePath("XObjComponent.js.ejs"),
```

Comment:
To be consistent with the template names, the file `XObjComponent.js.ejs` should be named `XComponent.js.ejs` instead.